presto-spark-launcher/src/main/java/com/facebook/presto/spark/launcher/PrestoSparkRunner.java

```diff
@@ -225,6 +229,7 @@ private IPrestoSparkService getOrCreatePrestoSparkService()
                             packageSupplier,
                             configProperties,
                             catalogProperties,
+                            Optional.empty(),
```

Comment:
Let's pass it. Don't forget to add the necessary checks similarly to other fields